lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -230,6 +246,44 @@ public function execute($params = null)
         return true;
     }
 
+    /**
+     * Handle $this->_longData after regular query parameters have been bound
+     *
+     * @throws MysqliException
+     */
+    private function _processLongData()
+    {
+        foreach ($this->_longData as $paramNr => $value) {
+            if (is_resource($value) && get_resource_type($value) === 'stream') {
+                $stream = $value;
+                while (! feof($stream)) {
+                    $chunk = fread($stream, 64 * 1024);
+                    if ($chunk === false) {
+                        throw new MysqliException("Failed processing the stream resource for parameter offset ${paramNr}.");
+                    }
+                    $this->_sendLongData($paramNr, $chunk);
+                }
+            } else {
+                $this->_sendLongData($paramNr, $value);
+            }
+        }
+    }
+
+    /**
+     * Bind parameters using send_long_data
+     *
+     * @param int    $paramNr  Parameter offset
+     * @param string $longData A chunk of data to send
+     *
+     * @throws MysqliException
+     */
+    private function _sendLongData($paramNr, $longData): void
```

Comment:
There's definitely something wrong with the CI, there [should be a space](https://github.com/doctrine/coding-standard/blob/1d35fd5/lib/Doctrine/ruleset.xml#L269) before the colon.